pokemongo_bot/cell_workers/spin_fort.py

```diff
@@ -40,9 +46,22 @@ def should_run(self):
             )
         return self.ignore_item_count or has_space_for_loot
 
+
     def work(self):
         forts = self.get_forts_in_range()
 
+        with self.bot.database as conn:
+            c = conn.cursor()
+            c.execute("SELECT DISTINCT COUNT(pokestop) FROM pokestop_log WHERE dated >= datetime('now','-1 day')")
+        if c.fetchone()[0] >= self.config.get('daily_spin_limit', 2000):
+           if not self.bypass_daily_limit:
+               self.emit_event('spin_limit', formatted='WARNING! You have reached your daily spin limit')
+               sys.exit(2)
+
+           if datetime.now() >= self.next_update:
+               self.emit_event('spin_limit', formatted='WARNING! You have reached your daily spin limit')
+               self._compute_next_update()
+
```

Comment:
Still needs to stop and return from this method. No point running the rest of the work code if daily_spin_limit is reached.